### ISSUE TITLE
[Usable- Make code - Choose a camera]: "Green background" text overlapping with button on applying text spacing.

### ISSUE THREAD

**Issue by Yamini234u** [microsoft/pxt-microbit#5504]
### **User Experience**
Users with cognitive disability and low vision who require increased space between lines, words, and letters will not be able to access the page efficiently If the text or the controls present on the page get overlapped or truncated.
**Note:**
User credentials should NOT be included in the bug.
### **Repro Steps:**
1.  Launch application URL : https://makecode.microbit.org
2. Home page will open.
3. Navigate "New Project" option and select it.
4.  Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Green Screen On" option and select it.
7. Verify that Green Screen setting is getting activated and the controls under "Choose a camera" dialogue. 
8. Apply text spacing
9. Verify "Green background" text overlapping with button or after on applying text spacing
### **Actual Result:**
"Green background" text overlapping with button on applying text spacing.
### **Expected Result:**
"Green background" text should not overlap with button on applying text spacing.

**ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:**
NA

**DOM details available:**
N/A
### **Reference Links:**
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [Link](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDc3NDY0MzExZDkxNTRlYmRhNzk2YTExYzE5Y2I2N2ZifDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3ODcwNDg4MTQ1NjM2NzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT0%3D&sdata=VHBCSFBzMTdmWExVNTk3MzZlTHdiME4rcFo3d3lyYk5kTTIvRjhLdTN3QT0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-chbhardwaj%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cv-dhti%40microsoft.com%7cf90880a914be401871f108d763233116%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637086876782361405&sdata=/ULS%2BbeqMjysCoiWHvFgAbSnuFJqxO3wtpHmlidPjbU%3D&reserved=0) for any process related queries.

### **Test Environment:**
- **OS Version:** Windows 11 23H2 (OS Build:22631.3007)
- **Browser:** Chrome Version 121.0.2277.4 (Official build) dev (64-bit).
- **Test Matrix:** Chrome + JAWS
- **URL:** https://makecode.microbit.org/.
<img width="1380" alt="Usable-Green background text overlapping with button on applying text spacing" src="https://github.com/microsoft/pxt-microbit/assets/117158108/ee394602-2e85-4ae0-8f46-f28fae1b1582">

**Comment by Yamini234u** [comment]
#Accessibility;#A11yUsable;#A11ySev4;#HCL;#E2E-HCL-MakeCode-Feb2024;#NonBenchmark;#Win11-Chrome;
